cli/pcluster/config/mappings.py

```diff
@@ -170,21 +180,27 @@
 }
 
 SCALING = {
-    "type": Section,
+    "type": CfnSection,
     "key": "scaling",
     "default_label": "default",
     "params": {
         "scaledown_idletime": {
-            "type": IntParam,
+            "type": IntCfnParam,
             "default": 10,
             "cfn_param_mapping": "ScaleDownIdleTime",
             "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP
-        }
+        },
+        "_scaledown_idletime": {
```

Comment:
While the "storage by composition" design would be perfectly suited for storage mechanisms like S3, where the persistence logic is uniquely defined for all the hierarchy, it wouldn't be of any benefit for the cfn parameters which have many different persistence logics. At the end, we would end up having a CfnStorage subclass for each CfnParameter subclass, making the mappings file more verbose and error prone since we would need to link each CfnParam with its proper storage manager.
Additionally, attaching multiple storage storage mechanisms to the same CfnParameter, despite being intuitively a good idea, would bring up complications like: 
- from which storage should this parameter be restored? 
- should storage mechanisms have a precedence? 
- Should we expose a new attribute to specify which is the 'main' storage to use"?
- How to deal with conflicting storage data?